site/app/views/NavigationView.php

```diff
@@ -1,10 +1,10 @@
 <?php
-
 namespace app\views;
-
 use \app\libraries\GradeableType;
+use app\models\Config;
 use app\models\Gradeable;
-
+use app\libraries\Core;
+//date_default_timezone_set($this->core->getConfig()->getTimezone());
```

Comment:
remove this commented out line. We'll never need to do this here. 

We can also probably get rid of the
```
use app\models\Config;
use app\libraries\Core;
```
lines.